src/isomorphic/classic/element/ReactElement.js

```diff
@@ -127,6 +127,10 @@ ReactElement.createElement = function(type, config, children) {
 
   if (config != null) {
     if (__DEV__) {
+      warning(
+        !(config.__proto__ != null && config.__proto__ !== Object.prototype),
+        'React.createElement(...): only instance properties will become props'
```

Comment:
I find this warning message a little bit confusing. Maybe we could write something like 


>React.createElement(...): Expected `props` argument to be a plain object. Props defined in its prototype chain will be ignored.

What do you think?